### PR TITLE
Org accnt updates

### PR DESCRIPTION
**Pull request by pdeguzman96**

### CODE REVIEW

File: app/web/api/organization_routes.py [hunk 1]
```diff
@@ -42,22 +42,26 @@ def get_organization_by_org_number(org_number, return_json=True):
     else:
         return result_dict
 
-def insert_organization(data, dt_now, add=True, flush=True):
+def insert_organization(data, dt_now, parent_org_id=None, org_type_id=4, add=True, flush=True):
```
Comment (201560543) [inline app/web/api/organization_routes.py:45]: Why is the org_type_id predefined here?

File: app/web/api/account_routes.py [hunk 2]
```diff
@@ -79,33 +83,42 @@ def create_new_account(return_json=True):
...
 
     # Insert Location Address (Ship to)
-    new_loc_addr = insert_address(data=data, org_id=new_org_id, dt_now=dt_now, org=False, add=True, flush=True)
+    new_loc_addr = insert_address(data=data, org_id=new_child_org_id, dt_now=dt_now, org=False, add=True, flush=True)
     current_app.logger.info('Inserted record:\n', new_loc_addr.as_dict())
     body['loc_address'] = new_loc_addr.as_dict()
     new_loc_addr_id = new_loc_addr.id  # Fetching address id of new record
 
-    # Insert Account
-    new_accnt = insert_account(data=data, org_id=new_org_id, loc_addr_id=new_loc_addr_id, dt_now=dt_now, add=True,
```
Comment (201560543) [inline app/web/api/account_routes.py:101]: We may have to insert the Position, phone number, and email into the new-account as well.

File: app/web/api/organization_routes.py (hunk 1 above)
Comment (pdeguzman96) [reply app/web/api/organization_routes.py:45]: hmm yeah no good reason to set it as default. I'll update

File: app/web/api/account_routes.py (hunk 2 above)
Comment (pdeguzman96) [reply app/web/api/account_routes.py:101]: Done

**Pull request merged**